File: src/main/java/com/jack/guess/Game.java
package com.jack.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Random random = new Random();
        int secret = random.nextInt(10) + 1;
        int number = 0;
        while(number != secret){
            System.out.print("Please enter the number:");
            number = scanner.nextInt();
            if(number > secret){
                System.out.println("Lower");
            } else if (number < secret){
                System.out.println("Higher");
            } else {
                System.out.println("anser is " + secret);
            }
        }
    }
}

File: src/main/kotlin/com/jack/Game.kt
package com.jack

import java.util.*

fun main() {
    val secret = Random().nextInt(10)+1
    //val scanner = Scanner(System.`in`)
    var number = 0
    while(number != secret){
        print("Please enter number :")
        //number = scanner.nextInt()
        number = readLine()!!.toInt()
        if(number > secret){
            println("Lower")
        } else if (number < secret){
            println("Higher")
        } else {
            println("answer is " + secret)
        }
    }
}